bluemira/codes/process/api.py

```diff
@@ -48,10 +49,15 @@ def __init__(self, filename):
         self.filename = filename
 
 
+class imp_data:  # noqa: N801, D101
+    __file__ = "./"
```

Comment:
Shouldn't `__file__` point to a _file_ path, rather than a directory? I'm not really sure what this `imp_data` class is doing, so I may be missing something.

Reply:
its a bit of a hack really as imp_data wont exist if we cant import process, I'll rework it a bit